docs/integrations/sources/referralhero.md

```diff
@@ -0,0 +1,28 @@
+# referralhero
+airbyte connector developed using airbyte 1.0 UI Connector development
```

Comment:
This should explain what the connector does, what referral hero is and what kind of data is synced, not _how_ it's built.

Reply:
Updated the Readme.md thanks

Reply:
Yeah, but why haven't you updated it here as well? ;-)